Calculator/Calculator.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.9.34728.123
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Calculator", "Calculator\Calculator.csproj", "{A75FD215-FD82-4FED-A014-2048FC14CF12}"
```

Comment:
solution refers to a project `Calculator\Calculator.csproj` however it was actually at `PinkSparrows.Calculator\Calculator.csproj`

Careful when submitting that the submitted solution runs.